src/main/java/org/teamresistance/frc/Gyro3D.java

```diff
@@ -0,0 +1,42 @@
+package org.teamresistance.frc;
+
+import com.kauailabs.navx.frc.AHRS;
+import edu.wpi.first.wpilibj.SPI;
+import org.strongback.components.AngleSensor;
+
+/**
+ * @author Shreya Ravi
+ */
+public class Gyro3D extends AHRS implements AngleSensor {
```

Comment:
Rather than inheriting from `AHRS`, favor the composition approach:

```java
private final AHRS gyro;

public Gyro3D(SPI.Port port) {
   this.gyro = ... // new AHRS here
}
```
Then, in all the other methods replace all references to `super` with `gyro`. This supports information hiding and encapsulation, and more importantly gives us full control over the API and lets us define how each method is meant to be used.

If you want, you can expose the AHRS like so:
```java
public AHRS getNavX() {
   return gyro;
}
```
However, this runs the risk of someone directly calling `getNavX().getYaw()` or something and not realizing it doesn't return normalized values, so I'd like to avoid this. With that said, it's still good to know we have this option in case we need the flexibility during build season.